tests/migrations/test_operations.py

```diff
@@ -38,6 +38,19 @@ def test_create_table() -> None:
             ),
         ).format_sql()
         == "CREATE TABLE IF NOT EXISTS test_table (id String, name Nullable(String), version UInt64) ENGINE ReplacingMergeTree(version) ORDER BY version SETTINGS index_granularity=256;"
+    ) or (
+        CreateTable(
+            StorageSetKey.EVENTS,
+            "test_table",
+            columns,
+            ReplacingMergeTree(
+                storage_set=StorageSetKey.EVENTS,
+                version_column="version",
+                order_by="version",
+                settings={"index_granularity": "256"},
+            ),
+        ).format_sql()
```

Comment:
Let's avoid duplicating all of this. You could rewrite this to:
```
assert CreateTable(...) in ["first_string", "second_string"]
```

Reply:
Oh yes that'd be much cleaner